digi/templates/digi/front_page.html

```diff
@@ -121,27 +121,28 @@ <h1 class="text-center">Ajankohtaista</h1>
           <div class="misc row">
               <div class="col-md-8">
                 <div class="blogs-preview">
-                  <div class="card-type">Blogin uusimmat</div>
-                  {% for blog in page.blog_posts|slice:":3" %}
+                  <div class="card-type">Uutisia</div>
+                  {% for feed in page.news_feeds|slice:":3" %}
                   <article class="blog-post">
                     <div class="row">
                     <div class="col-sm-5">
                       <div class="match-height clearfix">
-                        {% image blog.header_image max-1920x1080 as blog_image %}
-                        <a href="{% pageurl blog %}"><div class="blog-post-main-image" style="background-image: url('{{ blog_image.url }}')"></div></a>
+                        {% if feed.image %}
+                        <a href="{{ feed.link }}"><div class="blog-post-main-image" style="background-image: url('{{ feed.image }}');{% if feed.is_default_image %}background-position:top;background-size:auto;{% endif %}"></div></a>
+                        {% endif %}
                       </div>
                     </div>
                     <div class="col-sm-7">
                       <div class="match-height clearfix">
-                      <h3><a class="hyphenate" href="{% pageurl blog %}">{{ blog.title }}</a></h3>
-                      <span class="blog-section-tag">{{ blog.parent_title }}</span> <time itemprop="startDate" datetime="{{ blog.date|date:'Y-m-j' }}">{{ blog.date|date:"j.n.Y" }}</time>
-                      {{ blog.body|first_p|safe }}
+                      <h3><a class="hyphenate" href="{{ feed.link }}">{{ feed.title }}</a></h3>
+                      <span class="blog-section-tag">{{ feed.category }}</span> <time itemprop="startDate" datetime="{{ feed.published|convert_datetime|date:'Y-m-j' }}">{{ feed.published|convert_datetime|date:"j.n.Y" }}</time>
+                      {{ feed.parsed_description|first_p|truncatechars_html:280|safe }}
                       </div>
                     </div>
                     </div>
                   </article>
                   {% endfor %}
-                  <a href="blogikirjoitukset/" class="blogs-preview-read-all">Kaikki blogitekstit <span class="caret-right"></span></a>
+                  <a href="http://digi.hel.fi/uutiset" class="blogs-preview-read-all">Lisää uutisia <span class="caret-right"></span></a>
```

Comment:
Was a hardcoded URL necessary for some reason?

Reply:
This link was copied from the card, but it has been fixed now to relative URL in this PR. The news index page selection for the front page has been managed now in https://github.com/City-of-Helsinki/digihel/pull/192 so it will prevents hard-coding.